providers/aws/resources/aws_elb.go

```diff
@@ -73,9 +73,11 @@ func (a *mqlAwsElb) getClassicLoadBalancers(conn *connection.AwsConnection) []*j
 						map[string]*llx.RawData{
 							"arn":                  llx.StringData(fmt.Sprintf(elbv1LbArnPattern, regionVal, conn.AccountId(), convert.ToString(lb.LoadBalancerName))),
 							"listenerDescriptions": llx.AnyData(jsonListeners),
-							"dnsName":              llx.StringData(convert.ToString(lb.DNSName)),
-							"name":                 llx.StringData(convert.ToString(lb.LoadBalancerName)),
-							"scheme":               llx.StringData(convert.ToString(lb.Scheme)),
+							"dnsName":              llx.StringDataPtr(lb.DNSName),
+							"name":                 llx.StringDataPtr(lb.LoadBalancerName),
+							"scheme":               llx.StringDataPtr(lb.Scheme),
+							"vpcID":                llx.StringDataPtr(lb.VPCId),
+							"createdTime":          llx.TimeData(toTime(lb.CreatedTime)),
```

Comment:
nit: I think if this ever returns `nil` (which it probably will not for the *creation*, the time here will show up as `1001-01-01` as `toTime` initializes an empty time struct. Better to use `llx.TimeDataPtr` directly which will handle nil accordingly by not returning an empty value

Reply:
Thanks for the explaination and the fix @preslavgerchev